extensions/amp-form/0.1/amp-form.js

```diff
@@ -276,6 +278,11 @@ export class AmpForm {
       for (let i = 0; i < varSubsFields.length; i++) {
         this.urlReplacement_.expandInputValueSync(varSubsFields[i]);
       }
+
+      // If this wasn't a submission event, trigger form submit.
```

Comment:
Ok, I split this method to multiple ones for clarity and handling a submission event is now separate from handling a submission action. PTAL 👀